website/docs/logger.md

```diff
@@ -0,0 +1,35 @@
+---
+id: logger
+title: Logging
+sidebar_label: Logging
+---
+
+# Logging
+`Core-Lib`'s `Logging` decorator provides logging of data like `classname`, `function name` and `message`. `Logging` decorator also lets us set the level of provided by `logging` python.

```

Comment:
this is not what this decorator dose, please rephraze using grammerly with this points
- automaticly log function call to log 
- use the pythin buildin logger to log function calls
- can be customize (no need to specify we have the parameters below )